source/render/RenderManager.cpp

```diff
@@ -665,6 +666,19 @@ void RenderManager::rrTemporalMarkTile(Tile* curTile)
     ent->setVisible(bb);
 }
 
+void RenderManager::rrDetachEntity(GameEntity* curEntity)
```

Comment:
When you add new code, please avoid the curEntity and, more generally, cur...
Just use the simple name like GameEntity* entity.